### ISSUE TITLE
Support Jetstream

### ISSUE THREAD

**Issue by AbstractiveNord** [infinyon/nats-connector#5]
Currently there's no way to replace Kafka with NATS, because Jetstream persistence layer isn't supported.

**Comment by pinkforest** [comment]
yep - good idea - Jetstream is supported by the existing underlying library so it could be added

I could send a PR if someone would like to test unless already working on it ? :eyes:

**Comment by AbstractiveNord** [comment]
> yep - good idea - Jetstream is supported by the existing underlying library so it could be added
> 
> I could send a PR if someone would like to test unless already working on it ? 👀

Feel free to send any PR.
